### DISCUSSION TITLE
Registering users

### DISCUSSION THREAD

**Discussion by fpolli** [statamic/cms#10997] [General]
I see there is a registration form tag, but it requires an email field and a password confirmation field. Can you create a regular form to register users? I don't use a password confirmation field, instead I use viewable password fields. Also, I prefer not to have an "email" field because a person can have multiple email fields and I prefer to have a "login" field instead of "email." Can we configure the registration form to not require those fields, or use a self-created form?

**Comment by edalzell** [comment]
Use a self created form then, and your own controller (or livewire)

**Comment by jackmcdade** [comment]
I think i could be down for a configurable `require_confirmation` config tho, would be nice 👌

**Reply by fpolli** [reply]
That sounds like solicitation for a PR. I would love to write something but I wouldn't know how. It would be cool, though.
